src/Pim/Bundle/DataGridBundle/Normalizer/DateTimeNormalizer.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+namespace Pim\Bundle\DataGridBundle\Normalizer;
+
+use Akeneo\Component\Localization\Presenter\PresenterInterface;
+use Symfony\Component\Serializer\Normalizer\NormalizerInterface;
+
+/**
+ * Normalize a DateTime into an localized date
+ *
+ * @author    Marie Bochu <marie.bochu@akeneo.com>
+ * @copyright 2017 Akeneo SAS (http://www.akeneo.com)
+ * @license   http://opensource.org/licenses/osl-3.0.php  Open Software License (OSL 3.0)
+ */
+class DateTimeNormalizer implements NormalizerInterface
+{
+    /** @var NormalizerInterface */
+    protected $standardNormalizer;
+
+    /** @var PresenterInterface */
+    protected $presenter;
+
+    /**
+     * @param NormalizerInterface $standardNormalizer
+     * @param PresenterInterface  $presenter
+     */
+    public function __construct(NormalizerInterface $standardNormalizer, PresenterInterface $presenter)
+    {
+        $this->standardNormalizer = $standardNormalizer;
+        $this->presenter = $presenter;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function normalize($date, $format = null, array $context = [])
+    {
+        $stdProductValue = $this->standardNormalizer->normalize($date, 'standard', $context);
+
+        $stdProductValue = $this->presenter->present($stdProductValue, ['locale' => $context['data_locale']]);
+
+        return $stdProductValue;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function supportsNormalization($data, $format = null)
+    {
+        return $data instanceof \DateTimeInterface && 'datagrid' === $format;
```

Comment:
maybe we can do that, but if it's done here, we should it for all normalizers, and I won't do that on this PR :)